analytics/management/commands/populate_analytics_db.py

```diff
@@ -63,6 +63,8 @@ def handle(self, *args: Any, **options: Any) -> None:
         stream = Stream.objects.create(
             name='all', realm=realm, date_created=installation_time)
         recipient = Recipient.objects.create(type_id=stream.id, type=Recipient.STREAM)
+        stream.recipient = recipient
+        stream.save(update_fields=["recipient"])
```

Comment:
It might make sense to switch this logic to call `create_stream_as_needed`, maybe with an argument to backdate the `date_created`, rather than having this be a duplicate bit of code for doing that.